### DISCUSSION TITLE
Charging upto 0.4 SOC

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#4009] [Q&A]
Is there any way by which we could define an experiment like "Charge at 1 C until 2 A.h" or "Charge at 1 C until 0.4 SOC"?
In general terms, to provide capacity/ SOC as the termination condition?

**Comment by valentinsulzer** [comment]
Yes but you have to define what you mean by "0.4 SOC" yourself using custom termination, see https://docs.pybamm.org/en/stable/source/examples/notebooks/simulations_and_experiments/custom-experiments.html
